### ISSUE TITLE
Report to compare invoiced items to received stock

### ISSUE THREAD

**Issue by jniles** [third-culture-software/bhima#4380]
The patient financial activity report goes a long way to showing what the patient has been billed, what they have paid, and what they have received as stock.  However, for increased transparency, it would be nice to directly compare the billing and the invoicing.

Imagine the following scenario - the institution changes its policy for malaria cures.  Now, instead of 10 pills of quinine 300mg, it is 12 pills of quinine 300mg.  The pharmacist goes ahead and starts giving 12 pills to every patient, but the invoicing department didn't realize this and continues to bill for 10 pills total.  How does an administrator find out that what is being billed and what is being received are not the same thing?

I propose a new kind of report that attempts to join the inventory items billed on the stock consumed.  This would allow you to directly compare the items invoiced with the items distributed to the patient.  As a first design, it would look something like this:

![image](https://user-images.githubusercontent.com/896472/80473702-68f3b080-893e-11ea-9180-7dbcc3b81b7b.png)
_Fig 1:  Invoiced Items Compared to Stock Received_

Basically, for each invoice, we want to look up the stock items distributed to that invoice.  If the stock movement isn't linked to an invoice, we would show it in an "unassigned" table.

The table shows:
  1. The quantity of the item invoiced (from the invoice)
  2. The price of the item invoiced (from the invoice)
  3. The quantity of stock consumed (from the stock movement)
  4. The cost of the stock consumed (from the stock movement)
  5. The difference in quantity between the invoice and the stock movement.  If more stock is consumed than invoiced, we should highlight this in red.
  6. The difference in cost.  The we have consumed more value than we invoiced, we should highlight this in red.

**Comment by lomamech** [comment]
@kwilu claim
